apps/nameguard.io/src/app/docs/displayed-name/page.tsx

```diff
@@ -101,10 +107,7 @@ export default function DisplayedNamePage() {
             <div className="flex items-center justify-center">
               <TruncatedText
                 maxDisplayWidth={60}
-                text={
-                  getExampleDisplayedName(DisplayedNameExample.SHORT_MAX_WIDTH)
-                    .name
-                }
+                text={getExampleTruncatedText(TruncatedTextExample.LONG_TEXT)}
               />
             </div>
           </div>
```

Comment:
A few lines below this in the `<pre>` tag we should change `displayTooltipWhenNameOverflows` to `displayTooltipWhenTextOverflows`